home/static/home/hero.css

```
@@ -17,6 +17,12 @@ html, body {
   line-height: 1.5;
 }
 
+.hero-bg {
+  background-image: url('./imagem-apresentacao-dadosdefeira.png'); background-size:100%; background-repeat: no-repeat; background-position: center; background-origin: padding-box; 
```

É uma boa prática colocar todas as propriedades uma embaixo da outra, para facilitar a leitura.

```suggestion
    background-image: url('./imagem-apresentacao-dadosdefeira.png');
    background-size:100%;
    background-repeat: no-repeat;
    background-position: center;
    background-origin: padding-box; 
```